src/context/PortfolioProvider/PortfolioContext.tsx

```diff
@@ -171,6 +182,79 @@ export const PortfolioProvider = ({ children }: { children: React.ReactNode }) =
     })()
   }, [assetsById, chainAdapter, dispatch, wallet])
 
+  const txIds = useSelector(selectTxIds)
+  const txsById = useSelector(selectTxs)
+  const txHistoryStatus = useSelector(selectTxHistoryStatus)
```

Comment:
Can't we use `useAppSelector` here?

Reply:
we could yes. now that you've fixed the circular imports we could, but we don't need to here.

`useAppSelector` simply types state in the selector function when we need to pass a param to a selector, but we don't need to pass a param here, so we can use the plain one